### DISCUSSION TITLE
Hide min/max

### DISCUSSION THREAD

**Discussion by evilmouse69** [kalkih/mini-graph-card#1048] [General]
How do I hide the min/max reading when hovering over the graph?
By the documentation it looks like it will be "extrema" but this doesn't fix it. set everything that has the "hover" option in the documentation to "false" and still shows

![WhatsApp Image 2024-01-02 at 18 14 50](https://github.com/kalkih/mini-graph-card/assets/38438298/835dc013-b7b4-49d3-95ec-162562dd0419)

Also is there anyway to get the stage closer to the title?

**Comment by ildar170975** [comment]
Start with fixing the code.
These lines are misplaced.
![image](https://github.com/kalkih/mini-graph-card/assets/71872483/e168b8a0-53c7-49a2-a94d-5af84f1c7462)


Besides, you are confusing "min/max reading" with "extrema".
On your screenshot no extrema shown.
You should have meant "labels" instead.

**Reply by evilmouse69** [reply]
Thanks, just spotted that I looked at the code above the section in the documentation for that, resolved that issue

**Reply by ildar170975** [reply]
close if resolved
